cmd/adduser.go

```diff
@@ -37,7 +37,32 @@ func CreateAddUserCmd() *cobra.Command {
 		Short:        "Add an user to the account",
 		Args:         cobra.MaximumNArgs(1),
 		SilenceUsage: true,
-		Example:      params.longHelp(),
+		Example: `# Add user with a previously generated public key:
+nsc add user --name <n> --public-key <nkey>
+# Note: that unless you specify the seed, the key won't be stored in the keyring.'
+
+# Set permissions so that the user can publish and/or subscribe to the specified subjects or wildcards:
+nsc add user --name <n> --allow-pubsub <subject>,...
+nsc add user --name <n> --allow-pub <subject>,...
+nsc add user --name <n> --allow-sub <subject>,...
+
+# Set permissions so that the user cannot publish nor subscribe to the specified subjects or wildcards:
+nsc add user --name <n> --deny-pubsub <subject>,...
+nsc add user --name <n> --deny-pub <subject>,...
+nsc add user --name <n> --deny-sub <subject>,...
+
+# To dynamically allow publishing to reply subjects
```

Comment:
To dynamically allow publishing to reply subjects, this works well for service responders.

Reply:
Fixed!